recipes/spyder/meta.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: spyder
+  version: 2.3.9
+
+source:
+  fn:  spyder-2.3.9.zip
+  url: https://pypi.python.org/packages/fb/37/09b789dbe321894afe1657f11f0026a5ea79987f1c30176b92ee648b3faa/spyder-2.3.9.zip
+  md5: dd01e07a77123c128ff79ba57b97c1d7
+
+build:
+  number: 0
+  osx_is_app: True
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - pyqt
+    - rope
+    - pyflakes
+    - jedi >=0.8
+    - qtconsole
+    - nbconvert
+    - pygments
+    - sphinx
+    - pep8
+    - psutil
+    - python.app      [osx]
+
+app:
+  entry: spyder
+  icon: logo.png
+  summary: Scientific Python Development Environment
+  type: desk
+
+test:
+  commands:
+    - spyder -h
+  imports:
+    - spyderlib
+
+about:
+  home: https://github.com/spyder-ide/spyder
+  license: MIT
+  license_file: LICENSE
+  summary: Scientific PYthon Development EnviRonment
+  description: |
```

Comment:
Thanks for the link! I was going to ask how does it look on the navigator :smile: 

Bare in mind that most people here are not too interested in GUIs. So, like @jakirkham said, we won't be against adding it, but we won't enforce it either since it adds an extra burden on the recipe maintainers.